hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -240,7 +244,7 @@ public Response put(@PathParam("bucket") String bucketName,
       if (exception.getResult() == ResultCodes.INVALID_BUCKET_NAME) {
         throw newError(S3ErrorTable.INVALID_BUCKET_NAME, bucketName, exception);
       }
-      LOG.error("Error in Create Bucket Request for bucket: {}", bucketName,
+      LOG.debug("Error in Create Bucket Request for bucket: {}", bucketName,
```

Comment:
Nit: unrelated change.

```suggestion
      LOG.error("Error in Create Bucket Request for bucket: {}", bucketName,
```